acm/src/main/java/org/literacybridge/acm/api/IDataRequestResult.java

```diff
@@ -12,30 +10,18 @@
  *
  */
 public interface IDataRequestResult {
-	/**
-	 * Returns the root category which should be used to walk
-	 * the tree of categories. The root category is a virtual root
-	 * and should generally not be displayed by the UI.
-	 */
-	public abstract Category getRootCategory();
+    /**
+     * Returns the facet count for the passed-in category.
```

Comment:
"Facet" seems to be a key Lucene concept, but I can't find any definition or description of how that relates to or is used by the ACM.